recipes/tslearn/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "tslearn" %}
+{% set version = "0.1.10.8" %}
+{% set sha256 = "7b1e22f93c107884fb104d5d25d37eb8772a341aeab91b3a22ce6d860ef4cde8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy 1.11.*
+  run:
+    - python
+    - numpy >=1.11
+    - scipy
+    - scikit-learn
+
+test:
+  imports:
+    - tslearn
+    - tslearn.metrics
+
+about:
+  home: http://tslearn.readthedocs.io/
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Any chance we can get this upstreamed?

Reply:
Made a PR upstream https://github.com/rtavenar/tslearn/pull/25

Reply:
Thanks. Feel free to update in the feedstock on the next release whenever that happens.